src/viewer.js

```diff
@@ -4359,7 +4360,7 @@ class VolumeImageViewer {
           windowWidth,
           colormap: [
             [...segment.style.paletteColorLookupTable.data.at(0), defaultSegmentStyle.backgroundOpacity],
-            [...segment.style.paletteColorLookupTable.data.at(-1)]
+            ...segment.style.paletteColorLookupTable.data.slice(1)
```

Comment:
Can you explain this change?

Reply:
Instead of using only the first and last values of the palette as if all segmentations were binary, we use the whole palette for different pixel intensities in fractional segmentations.